ui/src/onboarding/containers/LoginPageContents.tsx

```diff
@@ -0,0 +1,385 @@
+// Libraries
+import React, {PureComponent, ChangeEvent, FormEvent} from 'react'
+import {
+  AlignItems,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  FlexBox,
+  FlexDirection,
+  Gradients,
+  Grid,
+  JustifyContent,
+  Method,
+  Notification,
+  Panel,
+  SelectGroup,
+} from '@influxdata/clockface'
+import auth0js from 'auth0-js'
+
+// Components
+import {LoginForm} from 'src/onboarding/components/LoginForm'
+import {SignUpForm} from 'src/onboarding/components/SignUpForm'
+import {SocialButton} from 'src/shared/components/SocialButton'
+import {GoogleLogo, GithubLogo} from 'src/clientLibraries/graphics'
+// Types
+import {Auth0Connection} from 'src/types'
+
+const auth0 = new auth0js.WebAuth({
```

Comment:
These values are simply placeholder values for a later iteration where config variables will be set by data obtained on IDPE